factory.py

```diff
@@ -101,7 +105,7 @@ def create_from_json(
 
         with open(path, "r") as f:
             args = json.load(f)
-
+        set_logger(logging_path=args['LOGGING_PATH'])
```

Comment:
why do we need to set the logger here?

Reply:
I have path issues when importing it in my factory so I included it in the main factory itself.

Reply:
yeah made similar change